app/Controllers/feedController.php

```diff
@@ -55,7 +55,7 @@ public static function addFeed(string $url, string $title = '', int $cat_id = 0,
 		$url = trim($url);
 
 		/** @var string|null $urlHooked */
-		$urlHooked = Minz_ExtensionManager::callHook('check_url_before_add', $url);
+		$urlHooked = Minz_ExtensionManager::callHook(Minz_HookType::CheckUrlBeforeAdd->value, $url);
```

Comment:
```suggestion
		$urlHooked = Minz_ExtensionManager::callHook(Minz_HookType::CheckUrlBeforeAdd, $url);
```

Suggestion, and for all similar calls. Requires function signature update. See below

Reply:
That was my ultimate goal. But I did not want to spend too much time if this was not the way to follow.